src/io.cpp

```diff
@@ -1241,6 +1320,12 @@ void PipelineComparatorAttitude(std::ostream &os,
     float attitudeErrorMean = attitudeErrorSum / expected.size();
     float fractionCorrect = (float)numCorrect / expected.size();
 
+    //printing values to console
+    std::cout << "\nAttitude Error Mean: " << attitudeErrorMean << "\n" << std::flush;
+    std::cout << "Attitude Number Correct " << numCorrect << "\n" << std::flush;
```

Comment:
shouldn't print out to `cout` -- it already prints to `os`, which the user can set, which is more correct behavior.